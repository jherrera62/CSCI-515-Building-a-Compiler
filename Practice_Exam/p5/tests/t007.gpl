//Incrementing a negative number. Shouldn't be any big deal.
int counter=-15;
on akey
{

  repeat {
    print("Counting up: " + counter);
    counter++;
  } until(counter > -10);

}
