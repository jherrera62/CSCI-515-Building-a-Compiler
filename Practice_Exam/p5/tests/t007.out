gpl.cpp::main()
  input file(tests/t007.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
int counter = -15
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().
print[7]: Counting up: -15
print[7]: Counting up: -14
print[7]: Counting up: -13
print[7]: Counting up: -12
print[7]: Counting up: -11
print[7]: Counting up: -10

User quit program. Printing the symbol table.
int counter = -9
